src/clio/subscriptions/SubscriptionManager.h

```diff
@@ -85,6 +87,8 @@ class SubscriptionManager
     boost::asio::io_context ioc_;
```

Comment:
Not in this PR, but does subscription manager need it's own io_context? Feel like we should just have two io_contexts. One for handling RPCs, and one for everything else.

Reply:
No, really it should probably use the `WorkQueue`. Or at least it should use the `workers` io_context.

It'd be nice to use the WorkQueue, because then publishing contributes to the count of the work the server is doing.